docker/mysql_tests/scripts/tests/binlog_push_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -e -x
+
+export WALE_S3_PREFIX=s3://mysqlbinlogpushbucket
+export WALG_MYSQL_DATASOURCE_NAME=sbtest:@/sbtest
+export WALG_MYSQL_BINLOG_DST=${MYSQLDATA}
+
+echo "CREATE DATABASE sbtest;" > /etc/mysql/init.sql
```

Comment:
Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.